### DISCUSSION TITLE
[Suggestion] Support for SSE/Pentium 3.

### DISCUSSION THREAD

**Discussion by apersonwhomakesstuff** [Alex313031/thorium-legacy#119] [Ideas]
im trying to install it but nothing works, thorium nope, supermium? nothing, dillo? NOT EVEN THAT WORKS and i just want something that works without having to install linux on a old windows xp pc

**Comment by leao-nardo** [comment]
that was 2 ys ago . so , did u got any browser working ?

**Reply by apersonwhomakesstuff** [reply]
that pc is gone now
